demo/repo/jenkins.groovy

```diff
@@ -1,7 +1,7 @@
 def jettyUrl = 'http://localhost:8081/'
 
 node('slave') {
-    git url: '/tmp/files/repo'
+    checkout scm
```

Comment:
More or less. A magic variable referring to the commit used by this version of `jenkins.groovy`.